### DISCUSSION TITLE
Project submission video length

### DISCUSSION THREAD

**Discussion by PriyaranjanKS** [microsoft/hack-together-dotnet#29] [Q&A] [answered]
@aaronpowell  @daisychaussee As part of the submission, could see that a demo video is required. Considering that we have numerous AI implementations the video length is stretching to around 15minutes. Is there a hard boundary limit on video length that will lead to disqualification ?

**Comment by aaronpowell** [comment, marked as answer]
We really would encourage people to be succinct in their video submissions and aim for around 5 minutes.
